jetty-io/src/main/java/org/eclipse/jetty/io/ByteBufferPool.java

```diff
@@ -252,6 +252,7 @@ int size()
 
         long getLastUpdate()
         {
+            // This could be changed to a getOpaque() call on JDK 9+.
```

Comment:
I think it should be `getAcquire()` because we want to see all modifications made by other threads (i.e. we want the semantic of a volatile read). I would remove the comment entirely -- the code difference would be enough.

Reply:
Sure, I'll remove the comment.